network/p2p/acp118/handler_test.go

```diff
@@ -143,18 +142,31 @@ func TestHandler(t *testing.T) {
 
 // The zero value of testVerifier allows signing
 type testVerifier struct {
-	Errs []*common.AppError
+	t            *testing.T
+	errs         []*common.AppError
+	clientNodeID ids.NodeID
```

Comment:
I prefer exporting these values because it makes refactoring easier. These aren't visible outside outside of the package anyways because they're in an unexported type + it's in a test file, but if we ever choose to move this into a testing file (say `acp118test` it makes the diff smaller for the future).